docs/source/index.rst

```diff
@@ -4,44 +4,32 @@ CSET Documentation
 .. toctree::
    :hidden:
 
+   self
+   getting-started/index
    usage/index
-   working-practices/index
-   operators
-   genindex
-   GitHub <https://github.com/MetOffice/CSET>
-
-:abbr:`CSET (Convective Scale Evaluation Tool)` is a tool to aid in verifying
-and evaluating convective-scale and turbulence-scale (regional and increasingly
-global) model configurations. It aims to replace the RMED RES and Toolbox and
-the collections of bespoke scripts littering people's home directories, reducing
-effort wasted on duplicating already existing code. This centralisation of
-diagnostics should also make evaluations more consistent, reproducible and
-comparable. Development takes place in the `CSET repository on GitHub`_.
+   reference/index
+   background/index
+   contributing/index
+
+:abbr:`CSET (Acronym meaning contested)` is a tool to aid in verifying and
+evaluating :abbr:`ESM (Earth System Model)` configurations. It aims to replace
+the :abbr:`RES (Regional Evaluation Suite)`, :abbr:`RMED (Regional Model
+Evaluation & Development)` Toolbox and the collections of bespoke scripts
+littering people's home directories, reducing effort wasted on duplicating
+already existing code. This centralisation of diagnostics should also make
```

Comment:
worth also commenting on improved quality assurance as well as preventing effort on duplicated code. I.e. adhoc scripts in different peoples dirs may have bugs, whereas CSET at least is scientific/technically reviewed code so more likely to catch bugs in diagnostics.

Reply:
Fixed with fff3fc4620b2138134805abcec31f093aec1c789